src/ducks/stripe.duck.js

```diff
@@ -509,7 +509,7 @@ export const createStripePaymentToken = params => dispatch => {
     })
     .catch(err => {
       const e = storableError(err);
-      dispatch(stripeAccountCreateError(e));
+      dispatch(stripePaymentTokenCreateError(e));
```

Comment:
One more thing: 
since this thunk function is not talking with our API/SDK but Stripe's SDK instead - `apiErrors` is not relevant. Just use "message" key from given [Error object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error).

Reply:
[apiErrors](https://github.com/sharetribe/flex-template-web/blob/master/src/util/errors.js#L219) is just our mapping on top of [errors array](https://www.sharetribe.com/api-reference/#errors) given by Marketplace API.

Reply:
You could also shorten some variables and function names since these are inside stripe.duck.js file (no need to repeat "stripe" word).
```js
//export const STRIPE_PAYMENT_TOKEN_CREATE_REQUEST = 'app/stripe/STRIPE_PAYMENT_TOKEN_CREATE_REQUEST';
export const CREATE_PAYMENT_TOKEN_REQUEST = 'app/stripe/CREATE_PAYMENT_TOKEN_REQUEST';
```
```js
// export const stripePaymentTokenCreateRequest = () => ({
export const createPaymentTokenRequest = () => ({
```

Here's an example of how I added `stripe.handleCardPayment` thunk call:
https://github.com/sharetribe/flex-template-web/pull/1090/files